src/components/sidebar/useEditChatTitle.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+import { useUpdateChatTitle } from '@/components/chats/useUpdateChatTitle'
+
+export const useEditChatTitle = (id: string, initialValue: string) => {
```

Comment:
I guess depends on how we want to do it. But not sure at the moment I'd need to work a little bit more because it's not a conventional structure and has many unique architecture decisions. We can keep everything in the same place but making use cases like we are it'll become quite hard to read really fast

Reply:
_We can keep everything in the same place but making use cases like we are it'll become quite hard to read really fast_
Let's talk